controllers/webhook/webhook.go

```diff
@@ -121,12 +121,9 @@ func (w *OpenDataHubWebhook) Handle(ctx context.Context, req admission.Request)
 		resp = w.checkDupCreation(ctx, req)
 	case admissionv1.Delete:
 		resp = w.checkDeletion(ctx, req)
-	default:
-		msg := fmt.Sprintf("No logic check by webhook is applied on %v request", req.Operation)
-		log.Info(msg)
-		resp = admission.Allowed("")
+	default: // for other operations by default it is admission.Allowed("")
+		// no-op
```

Comment:
Should it initialize  resp then? Because resp.Allowed is bool I guess the default value will be `false`. Will not affect real situation, we should not hit that branch, but anyway.